modules/behavioural_qc/php/provisioners/incompleteprovisioner.class.inc

```diff
@@ -0,0 +1,101 @@
+<?php declare(strict_types=1);
+/**
+ * PHP version 7
+ *
+ * @category Behavioural
+ * @package  Loris
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ *           Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+namespace LORIS\behavioural_qc\Provisioners;
+
+/**
+ * The IncompleteProvisioner class
+ *
+ * PHP version 7
+ *
+ * @category Behavioural
+ * @package  Loris
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ *           Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+class IncompleteProvisioner extends \LORIS\Data\Provisioners\DBObjectProvisioner
+{
+    /**
+     * Create a RowProvisioner
+     */
+    function __construct()
+    {
+        // Note: Need to modify how query is built.
```

Comment:
I don't understand what you're trying to modify or why. Can you elabourate on this comment?

Reply:
@driusan old comment. I'll remove.